scripts/host-testbench.sh

```diff
@@ -44,6 +44,55 @@ function srcsize() {
   echo "$OUTPUT_SIZE"
 }
 
+function test_comp() {
+    CMD=$1
+    COMP=$2
+    LOGFN=${COMP}_log.txt
+    OUTFN=${COMP}_out.raw
+    echo "=========================================================="
+    echo "test $COMP with ${CMD}"
+    if ${CMD} &> $LOGFN; then
+	echo "$COMP run success"
+    else
+	echo "$COMP run failed!"
+	cat $LOGFN
+	exit 1
+    fi
+    if comparesize ${INPUT_FILE_SIZE} "$(filesize $OUTFN)"; then
+	echo "$OUTFN size check passed!"
+    else
+	echo "$OUTFN size check failed!"
+	cat $LOGFN
+	exit 1
+    fi
+    rm $OUTFN $LOGFN
+}
+
+function test_src_comp() {
+    CMD=$1
+    COMP=$2
+    LOGFN=${COMP}_log.txt
+    OUTFN=${COMP}_out.raw
+    echo "=========================================================="
+    echo "test $COMP with ${CMD}"
+    if ${CMD} &> $LOGFN; then
+	echo "$COMP run success"
+    else
+	echo "$COMP run failed!"
+	cat $LOGFN
+	exit 1
+    fi
+
+    if comparesize "$(srcsize ${INPUT_FILE_SIZE} $3 $4)" "$(filesize $OUTFN)";then
```

Comment:
do I see it correctly, that the only difference between these two functions is the first parameter of `comparesize`? Maybe they could be further unified then to be called `x(cmd, comp, ${INPUT_FILE_SIZE})` and `x(cmd, comp, $(srcsize ${INPUT_FILE_SIZE} $3 $4))`?

Reply:
Maybe I'll do another PR for test case increase for new components, @marc-hb proposed that this PR would be min changes only to support the additional testbench command line parameters. So this script would not change at all.

Reply:
Thanks, I will use this idea later when I add more components to default test in CI